cardano-node/src/Cardano/Node/Configuration/Topology.hs

```diff
@@ -35,98 +36,83 @@ import           Ouroboros.Consensus.Util.Condense (Condense (..))
 
 -- | A data structure bundling together a node identifier and the path to
 -- the topology file.
-data TopologyInfo = TopologyInfo {
-    node         :: NodeId
-  , topologyFile :: FilePath
-  }
+data TopologyInfo = TopologyInfo {node :: NodeId, topologyFile :: FilePath}
 
 -- | IPv4 address with a port number.
---
-data NodeAddress = NodeAddress {
-      naHostAddress :: !IP.IP
-    , naPort        :: !PortNumber
-    }
+data NodeAddress = NodeAddress {naHostAddress :: !IP.IP, naPort :: !PortNumber}
   deriving (Eq, Ord, Show)
 
 instance Condense NodeAddress where
-    condense NodeAddress {naHostAddress, naPort}
-      = show naHostAddress ++ ":" ++ show naPort
+  condense (NodeAddress addr port) = show addr ++ ":" ++ show port
 
 instance FromJSON NodeAddress where
-    parseJSON = withObject "NodeAddress" $ \v -> do
-      NodeAddress
+  parseJSON = withObject "NodeAddress" $ \v -> do
+    NodeAddress
       <$> (read <$> v .: "addr")
       <*> ((fromIntegral :: Int -> PortNumber) <$> v .: "port")
 
-nodeAddressToSockAddr
-    :: NodeAddress
-    -> SockAddr
-nodeAddressToSockAddr NodeAddress {naHostAddress, naPort} = case naHostAddress of
-    IP.IPv4 ipv4 -> SockAddrInet  naPort   (IP.toHostAddress  ipv4)
-    IP.IPv6 ipv6 -> SockAddrInet6 naPort 0 (IP.toHostAddress6 ipv6) 0
+nodeAddressToSockAddr :: NodeAddress -> SockAddr
+nodeAddressToSockAddr (NodeAddress addr port) =
+  case addr of
+    IP.IPv4 ipv4 -> SockAddrInet port $ IP.toHostAddress ipv4
+    IP.IPv6 ipv6 -> SockAddrInet6 port 0 (IP.toHostAddress6 ipv6) 0
 
 nodeAddressInfo :: NodeAddress -> AddrInfo
-nodeAddressInfo na@NodeAddress {naHostAddress}
-    = AddrInfo {
-        addrFlags      = addrFlags defaultHints
-      , addrFamily     = case naHostAddress of
-                          IP.IPv4{} -> AF_INET
-                          IP.IPv6{} -> AF_INET6
-      , addrSocketType = Stream
-      , addrProtocol   = addrProtocol defaultHints
-      , addrAddress    = nodeAddressToSockAddr na
-      , addrCanonName  = Nothing
-    }
+nodeAddressInfo na@(NodeAddress hostAddr _) =
+  AddrInfo { addrFlags = addrFlags defaultHints
```

Comment:
I wonder if this way of formatting things is prone to reindentation, if `AddrInfo` is ever renamed -- although in this particular case that seems unlikely.

Reply:
Not sure how likely it will be renamed but, putting the fields on a newline is an easy fix :slightly_smiling_face: